orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -83,7 +82,17 @@ class RedisQueue(
         ?.also { id ->
           val ack = this::ackMessage.partially1(id)
           redis.readMessage(id) { message ->
-            callback(message, ack)
+            val attempts = message.getAttribute(AttemptsAttribute::class.java)!!.attempts
```

Comment:
Refactored it and turf'd the `!!`.

The goal would be have some reasonably flexible metadata ... @robzienert has thoughts around annotating message w/ info that can be used when throttling / shaping traffic.

Maybe it makes sense to store `delay` as an attribute and favor using that in the event that a message is retried/redriven? (perhaps if a caller does not explicitly provide an overridden value when doing a `.push()`.